docs/rules/no-children-prop.md

```diff
@@ -0,0 +1,21 @@
+# Prevent passing of children as props (no-children-prop)
+
+Children should always be actual children, not passed in as a prop.
+
+## Rule Details
+
+The following patterns are considered warnings:
+
+```js
```

Comment:
That does look better :-) either way's fine, but i'm pretty sure it's come up before. Maybe Github improved the highlighting since.